src/lib/homestore_backend/index_kv.cpp

```diff
@@ -74,7 +81,7 @@ BlobManager::Result< homestore::MultiBlkId > HSHomeObject::move_to_tombstone(sha
     BlobRouteValue index_value_get;
     homestore::BtreeSingleGetRequest get_req{&index_key, &index_value_get};
     auto status = index_table->get(get_req);
-    if (status != homestore::btree_status_t::success) {
+    if (status != homestore::btree_status_t::success && recovery_done_) {
         LOGDEBUG("Failed to get from index table [route={}]", index_key);
```

Comment:
Is there a reason this is not a LOGERROR? 

Since the caller is also checking recovery_done_, can we keep this function out of the knowledge from recovery? maybe I didn't get the reason why both the caller and this function needs to check it.

Reply:
We didnt want to print those errors during recovery. as we are ignoring them.